### PR TITLE
Add e2e test for global lp pool

### PR DESCRIPTION
**Pull request by umangyadav**
https://github.com/llvm/torch-mlir/pull/3435

CC: @zjgarvey  @vivekkhandelwal1

### CODE REVIEW

File: e2eshark/onnx/operators/GlobalLpPool/model.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+print("Output shape:", numpy.array(model_output[0]).shape)
+
+# Moving to torch to handle bfloat16 as numpy does not support bfloat16
+E2ESHARK_CHECK["input"] = [torch.from_numpy(model_input_X)]
+E2ESHARK_CHECK["output"] = [torch.from_numpy(arr) for arr in model_output]
+
+print("Input:", E2ESHARK_CHECK["input"])
+print("Output:", E2ESHARK_CHECK["output"])
+
+# TEST(2) : Non-Default p value 2D case
```
Comment (umangyadav) [inline e2eshark/onnx/operators/GlobalLpPool/model.py:83]: Test for https://github.com/llvm/torch-mlir/pull/3460

**Review by vivekkhandelwal1 — APPROVED** [review]
LGTM

**Comment by umangyadav** [comment]
@vivekkhandelwal1  these tests have dependency on https://github.com/llvm/torch-mlir/pull/3460. That PR needs to get merged first.

**Comment by vivekkhandelwal1** [comment]
> llvm/torch-mlir#3460

Merged that PR.

**Pull request merged**